Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1529,4 +1529,5 @@ class RebootWithUpdatedCrossTargetImage(shell.ShellCommandNewStyle):
     @defer.inlineCallbacks
     def run(self):
         rc = yield super().run()
+        self.build.buildFinished(['Rebooting with updated image, retrying build'], RETRY)
```

Comment:
Optional: Can change the next line to `defer.returnValue(SUCCESS)` as well, since the step itself was successful and did what it is expected to do.